### PR TITLE
Update [Examples]-Create-and-Assign-Custom-RBAC-Roles.md

### PR DESCRIPTION
**Pull request by mbilalamjad**
## Overview/Summary
This PR is meant to provide details on a scenario where a role assignment needs to be done on an SPN. This is in response to the following issue raised by one of our community members
https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues/453#issuecomment-1238318792

## This PR fixes/adds/changes/removes

1. The PR add guidance on how to assign role to SPN

### Breaking Changes

1. None

## Testing Evidence

Testing evidence is available in the issue mentioned in which Kevin shared the tested steps to assign SPN a role.

## As part of this Pull Request I have

- [x] Checked for duplicate [Pull Requests](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/pulls)
- [x] Associated it with relevant [issues](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues), for tracking and closure.
- [x] Ensured my code/branch is up-to-date with the latest changes in the `main` [branch](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/tree/main)
- [x] Performed testing and provided evidence.
- [x] Updated relevant and associated documentation.
- [x] Updated the ["RELEASE"](https://github.com/Azure/terraform-azurerm-caf-enterprise-scale/issues) page with details needed for when this code change is released.

### CODE REVIEW

File: docs/wiki/[Examples]-Create-and-Assign-Custom-RBAC-Roles.md [hunk 1]
```diff
@@ -103,6 +103,8 @@ If you don't already have an `archetype_extension_es_landing_zones.tmpl.json` fi
 This code will assign your new `Reader-Support-Tickets` role to a group named `Contoso Reader and Support Tickets`.
 In order to assign the `Reader-Support-Tickets` role to the group, you need to use the groups objectID which can be located in Azure Active Directory.
 
+Instead of group, if you would like to assign the role to a SPN in the form of application registered in Azure AD then make sure to use the Object ID of the Service Principal related to that application which you can find from the "Enterprise applications" blade of Azure AD Portal. For more details on the relation between application and service principal objects please see [this article](https://learn.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals).
```
Comment (matt-FFFFFF) [inline docs/wiki/[Examples]-Create-and-Assign-Custom-RBAC-Roles.md:106]: Please can you take the locale out of the URL?

**Review by matt-FFFFFF — CHANGES_REQUESTED** [review]
Thanks for this pr. please can you fill out the pr details too?

File: docs/wiki/[Examples]-Create-and-Assign-Custom-RBAC-Roles.md (hunk 1 above)
Comment (mbilalamjad) [reply docs/wiki/[Examples]-Create-and-Assign-Custom-RBAC-Roles.md]: Thanks put for the feedback, updated the PR accordingly

**Review by matt-FFFFFF — APPROVED** [review]

**Pull request merged**